tests/storage/hotplug.go

```diff
@@ -955,9 +955,14 @@ var _ = SIGDescribe("Hotplug", func() {
 				libwait.WaitForSuccessfulVMIStart(vmi,
 					libwait.WithTimeout(240),
 				)
-				testVolumes := make([]string, 0)
-				dvNames := make([]string, 0)
-				for i := 0; i < 75; i = i + 5 {
+
+				const howManyVolumes = 75
```

Comment:
`numVolumes`?

Reply:
why not, naming is always tricky :)

Reply:
Actually, let the CI spin since it already started testing, I think it's anyway pretty clear what the const means, is that ok?